pkg/providers/instancetype/types.go

```diff
@@ -350,10 +351,13 @@ func ENILimitedPods(ctx context.Context, info *ec2.InstanceTypeInfo) *resource.Q
 	return resources.Quantity(fmt.Sprint(usableNetworkInterfaces*(addressesPerInterface-1) + 2))
 }
 
-func privateIPv4Address(info *ec2.InstanceTypeInfo) *resource.Quantity {
+func privateIPv4Address(name string) *resource.Quantity {
```

Comment:
nit: instancetype or instancetypename
```suggestion
func privateIPv4Address(instanceType string) *resource.Quantity {
```

Reply:
Shore